<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportPlotDialog</class>
 <widget class="QDialog" name="ExportPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>551</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Create in:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="folderNameLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="browsePushButton">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Browse ..</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>551</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>File name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fileNameLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>File format:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fileTypeComboBox">
      <item>
       <property name="text">
        <string>Pdf</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Png</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Jpg</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bmp</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>133</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="widthSpinBox">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>px</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>35</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="heightSpinBox">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>px</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportPlotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportPlotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
